scripts/build/package.ts

```diff
@@ -87,7 +101,14 @@ function main() {
             const packageJson: typeof manifest = JSON.parse(fs.readFileSync(packageJsonFile, { encoding: 'utf-8' }))
             const versionSuffix = getVersionSuffix()
             const version = packageJson.version
-            packageJson.version = args.debug ? `1.99.0${versionSuffix}` : version.replace('-SNAPSHOT', versionSuffix)
+            if (isBeta()) {
+                // Setting the version to an arbitrarily high number stops VSC from auto-updating the beta extension
+                packageJson.version = `1.999.0${versionSuffix}`
+            } else {
+                packageJson.version = args.debug
+                    ? `1.99.0${versionSuffix}`
```

Comment:
should this also be 1.999 ? The inconsistency could raise questions for future readers.